third_party/system_libraries.BUILD

```diff
@@ -43,3 +43,32 @@ cc_library(
         ],
     }),
 )
+
+cc_library(
```

Comment:
Can we add these dependencies to the bazel-base Dockerfile too?

Reply:
We can do that - but does bazel-base have a purpose any longer?

Reply:
I still like it for building out new bazel files since it only has dependencies that we need. I think eventually we want our devcontainer docker image to be as clean as the bazel base one.